exporter/clickhouselogsexporter/exporter.go

```diff
@@ -342,7 +342,9 @@ func attributesToSlice(attributes pcommon.Map, forceStringValues bool) (response
 				response.FloatValues = append(response.FloatValues, v.Double())
 			case pcommon.ValueTypeBool:
 				// add boolValues in future if it is required
-				response.BoolKeys = append(response.BoolKeys, formatKey(k))
+				// store it as a string in clickhouse for now
+				response.StringKeys = append(response.StringKeys, formatKey(k))
+				response.StringValues = append(response.StringValues, v.AsString())
```

Comment:
> Added changes so that the bool values are stored


I am trying to understand why/where are they dropped currently and why do we need to store them as string?

Reply:
Also this had related bool support and different implementation https://github.com/SigNoz/signoz-otel-collector/pull/183

Reply:
So they were dropped because previously we stored them in the string key values array. But after adding tag attributes we added bool but it was not actually stored with the log line.

yeah adding the materialized table will be better with new column. Let me make the changes.